pillar/api-gateway-public.sls

```diff
@@ -206,7 +206,7 @@ api_gateway:
     consumers:
         # user: key
         anonymous: public
-        # elife-xpub: secretkey
+        # elife-bot: secretkey
```

Comment:
This is an odd switch? Was it intentional?

Reply:
Again, not that important as it's still commented out. But seems odd that it switched from `xpub` to `bot`.

Reply:
ah, it is an odd switch. There is no logic behind it other than "not xpub".

I'm doing as the Egyptians did and scrubbing xpub from all written history.

Reply:
just to elaborate a little, my process for tearing down a project is to find all mentions of that project and ensure they are properly tied off. This includes comments, configuration, secrets, issues, credentials, etc - they should all be cleaned up.

Our process hasn't been particularly strict in the past and this has led to the accumulation of 'stuff' that we become afraid of removing as we encounter it in the future in case it breaks something.

Our future selves just want to get their job done quickly and with little fuss or stress and that means keeping the paths clear and obvious. It's difficult to quantify how worthwhile this is in the *present* when we're not tripping over it in the *future*.